docs/source/running-jobs/distributed-jobs.rst

```diff
@@ -54,3 +60,27 @@ SkyPilot exposes these environment variables that can be accessed in a task's ``
     :code:`run` command with :code:`echo $SKY_NODE_IPS >> ~/sky_node_ips`.
 - :code:`SKY_NUM_GPUS_PER_NODE`: number of GPUs reserved on each node to execute the
   task; the same as the count in ``accelerators: <name>:<count>`` (rounded up if a fraction).
+
+
+Executing a multi-node task
+-----------------------------------------
+
+The execution behavior is: all nodes are provisioned (barrier);
+workdir/file_mounts are synced to all nodes (barrier); ``setup`` commands are
+executed on all nodes (barrier); finally, the ``run`` commands are executed on
+all nodes.
+
+
+To execute a command on the head node only (common for tools like ``mpirun``),
+use ``SKY_NODE_RANK`` as follows:
+
+.. code-block:: yaml
```

Comment:
I am for removing this entire section (executing a multi-node task).

We should the code snippet, to to the above section (pytorch-distributed-resnet), and say if the user only runs to run on the head node, use this:

```
run: |
     if [ "${SKY_NODE_RANK}" == "0" ]; then
         # Launch the head-only command here.
     fi
```

Reply:
This is important since it's prompted by a user.